addons/payment/models/payment_acquirer.py

```diff
@@ -98,9 +98,11 @@ def _get_providers(self, cr, uid, context=None):
     }
 
     _defaults = {
+        'provider': 'transfer',
```

Comment:
Set this default value in payment_transfer module. payment_transfer is automatically installed when payment is installed.